app/templates/components/footer-main.hbs

```diff
@@ -101,7 +101,7 @@
         </div>
       </div>
     </div>
-    <div class="ui inverted section divider"></div>
+    <div class="ui inverted section divider"style="margin-bottom: 40px!important;"></div>
```

Comment:
mb-4

Reply:
> mb-4

 I added the code can you review

Reply:
It's not working AFAIK

Reply:
already updated the code.

Reply:
alrredy update the code but,

@iamareebjamal can you tell me what is the purpose of writing the i am working on this issue if some gets solutions can make pr on any issue it means we are wasting out time

Reply:
Can you tell me what's the purpose of reviewing PRs and writing "It's not working AFAIK" and not getting any follow up on that?